scripts/linters/js_ts_linter.py

```diff
@@ -1117,6 +1117,41 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from eslint messages.
+
+        Args:
+            lint_messages: str. Messages returned by the js_ts linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        error_messages = ''
+        # Extract the message from list and split the message by newline
+        # so that we can use them and remove last four lines from the end.
+        # Becuase last two lines are empty with a newline character and
+        # third one have a message with number of errors.
+        # 1 errors found in 1 file
+        # and fourth one with a message to fix automatically.
+        # These error can be fixed with the use of --fix.
+        messages = lint_messages.split('\n')[:-4]
+        for message in messages:
+            # ESlint messages start with line numbers and then a
+            # cross(x) and a message-id in the end. We are matching
+            # if the line contains line number becuase every message start with
+            # num:num where num is of type int and we are matching it with regex
+            # and if that is True then we are removing cross(x) which is at the
+            # index 1 and message-id from the end.
+            if re.search(r'^\d+:\d+', message.lstrip()):
```

Comment:
Comments above apply here too.

I don't think you necessarily need a new method if you can encode all the necessary information in the regex. In general you shouldn't be doing stuff with `:1`, `2:-1`, etc. -- hard to understand what the numbers mean.